### ISSUE TITLE
linker errors

### ISSUE THREAD

**Issue by theGrumpiestSteve** [standard-ai/fast_fp#1]
attempting to build a small project including fast_fp on Ubuntu 22.04 with rustc 1.62.0 (stable), I get the following error at the very end of the build process

/sr/bin/ld: /home/steve/filter2/target/release/deps/libfast_fp-xxxxxxxxx.rlib: error adding symbols: file format not recognized.  

I get the same when I try to build with the nightly version of the compiler.

**Comment by rnarubin** [comment]
Can you share the command you use to build your project please

**Comment by theGrumpiestSteve** [comment]
I could, although last night I had an epipany:  the build system was using the standard gnu linker, but it really needed to use the llvm linker (lld).  Although I had installed LLVM, I hadn't replaced gnu linker by the llvm one.  Once I had corrected that it all built quite happily.  In my defence, I don't think any of the Rust documentation tells me that I have to use the LLVM linker (if it does then, clearly, I ought to read it again!).  Thank you.
